paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -134,7 +135,12 @@ internal class PaymentOptionsViewModel @Inject constructor(
 
         updateSelection(args.state.paymentSelection)
 
-        isEligibleForCardBrandChoice = args.state.isEligibleForCardBrandChoice
+        cbcEligibility = when (args.state.isEligibleForCardBrandChoice) {
+            true -> CardBrandChoiceEligibility.Eligible(
+                preferredNetworks = args.state.config?.preferredNetworks ?: listOf()
+            )
+            false -> CardBrandChoiceEligibility.Ineligible
+        }
```

Comment:
We can move this mapping into `PaymentSheetLoader` so that the state only ever holds the `CardBrandChoiceEligibility` class. Can be done in a follow-up.

Reply:
Will do in a follow-up since a lot of test methods use the `PaymentSheetState.Full` at the moment.